subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -114,6 +114,15 @@ include::sample[dir="snippets/dependencyManagement/catalogs-settings/groovy",fil
 include::sample[dir="snippets/dependencyManagement/catalogs-settings/kotlin",files="build.gradle.kts[tags=use_version]"]
 ====
 
+If the alias of a declared version is also a prefix of some more specific alias, as in `version("zinc", "...")"` and `version("zinc-apiinfo", "...")`, then
```

Comment:
I would use version numbers instead of ellipsis in the example. Or focus on the generated side, that is you end up with `zinc` and `zinc.apiinfo`.